### PR TITLE
add install log regexps for terraform inconsistent apply (a known intstaller bug)

### PR DESCRIPTION
**Pull request by gregsheremeta**
/assign @joelddiaz 
/assign @2uasimojo

### CODE REVIEW

File: pkg/controller/clusterprovision/installlogmonitor_test.go [hunk 1]
```diff
@@ -47,6 +47,7 @@ const (
...
 	proxyInvalidCABundleLog   = "time=\"2021-08-27T05:56:50Z\" level=info msg=\"[pull.q1w2.quay.rhcloud.com/openshift-release-dev/ocp-release@sha256:7047acb946649cc1f54d98a1c28dd7b487fe91479aa52c13c971ea014a66c8a8: error pinging docker registry pull.q1w2.quay.rhcloud.com: Get \\\"https://pull.q1w2.quay.rhcloud.com/v2/\\\": proxyconnect tcp: x509: certificate signed by unknown authority]): quay.io/openshift-release-dev/ocp-release@sha256:7047acb946649cc1f54d98a1c28dd7b487fe91479aa52c13c971ea014a66c8a8: error pinging docker registry quay.io: Get \\\"https://quay.io/v2/\\\": proxyconnect tcp: x509: certificate signed by unknown authority"
+	inconsistentResult        = "time=\"2021-12-01T16:08:46Z\" level=error msg=\"Error: Provider produced inconsistent result after apply\""
```
Comment (2uasimojo) [inline pkg/controller/clusterprovision/installlogmonitor_test.go:50]: Can you please add a test case that uses this?

Comment (gregsheremeta) [reply pkg/controller/clusterprovision/installlogmonitor_test.go:50]: oops! done

**Comment by 2uasimojo** [comment]
Since this needs a rebase anyway...

> (a known installer bug)

Should we include a comment pointing to the bug? Presumably once the bug gets resolved (and vendored in?) OSD would no longer want to suppress this error if it pops up?

**Comment by gregsheremeta** [comment]
> OSD would no longer want to suppress this error if it pops up?
My plan is to make this one of the retryable conditions. If installer fixes the bug, I still think I'd want to retry if we ever saw it again. (I'd also want to alert on it somehow, but that's not Hive's concern.)

> Should we include a comment pointing to the bug?
Sure. I'll have to track it down though.

File: config/configmaps/install-log-regexes-configmap.yaml [hunk 2]
```diff
@@ -199,6 +199,11 @@ data:
 
 
     # Generic OpenShift Install
+    - name: InconsistentResult
+      searchRegexStrings:
+      - "Error: Provider produced inconsistent result after apply"
+      installFailingReason: InconsistentResult
```
Comment (joelddiaz) [inline config/configmaps/install-log-regexes-configmap.yaml:205]: InconsistentResult sounds a little generic. Can we call it InconsistentTerraformResult?

Comment (gregsheremeta) [reply config/configmaps/install-log-regexes-configmap.yaml]: done

**Comment by gregsheremeta** [comment]
> I'll add it when I do the rebase

:heavy_check_mark:

**Pull request merged**